src/Nest/Search/Suggesters/Suggest.cs

```diff
@@ -14,5 +15,10 @@ public class Suggest
 		public string Text { get; internal set; }
 		[JsonProperty("options")]
 		public IEnumerable<SuggestOption> Options { get; internal set; }
+
+		public IEnumerable<CompletionOption<T>> GetCompletionOptions<T>()
+		{
+			return Options.Select(x => new CompletionOption<T>(x));
```

Comment:
Could use some C# 6 love, `Options?.Select(x => new CompletionOption<T>(x)) ?? Enumerable.Empty<CompletionOption<T>>();` and can be an expression bodied method :+1: